apps/src/templates/instructions/ThreeColumns.jsx

```diff
@@ -57,4 +58,4 @@ ThreeColumns.propTypes = {
   }
 };
 
-export default ThreeColumns;
+export default Radium(ThreeColumns);
```

Comment:
I think this is actually because one of the children (CollapserButton) passed to this component uses an array of styles, and it wasn't working without ThreeColumns being Radium enabled.